### PR TITLE
Added references to changelog maker tool and other minor fixes

### PR DESCRIPTION
**Pull request by NickNaso**
I added some reference about which tool use to generate the changelog and other minor fixes.

### CODE REVIEW

File: doc/creating_a_release.md [hunk 1]
```diff
@@ -1,30 +1,51 @@
...
+following the style used in publishing the previous release.
+
+* Add any new contributors to the "contributors" section in the package.json
+
+* Validate all tests pass by running npm test on master.
 
-* Add any new contributors to the "contributors" section in
-  the package.json
+* Use **[CI](https://ci.nodejs.org/view/x%20-%20Abi%20stable%20module%20API/job/node-test-node-addon-api/)**
+to validate tests pass for latest 11, 10, 9, 8, 6, 4 releases (note there are still some issues on SmartOS and
```
Comment (mhdawson) [inline doc/creating_a_release.md]: I think we can remove 4 from the list, as well as 9 which is no longer supported.

Comment (NickNaso) [reply doc/creating_a_release.md]: Done!

Comment (addaleax) [reply doc/creating_a_release.md]: @mhdawson @NickNaso Are these still in CI? I would not drop them from the CI runs without a semver-major bump of the package (at least for v4.x)…

Comment (NickNaso) [reply doc/creating_a_release.md]: Hi @addaleax I don't know too much about CI, @mhdawson could you answer to question asked by @addaleax?

Comment (mhdawson) [reply doc/creating_a_release.md]: What we have in our N-API CI is: 6, 8, 10 here:

https://ci.nodejs.org/view/x%20-%20Abi%20stable%20module%20API/job/node-test-node-addon-api-LTS%20versions/

and

v11 here:

https://ci.nodejs.org/view/x%20-%20Abi%20stable%20module%20API/job/node-test-node-addon-api/

I should change it to include V12 as well.

These run nightly.

**Comment by NickNaso** [comment]
What do I need to change here? Someone could review?

**Comment by mhdawson** [comment]
I think it just needs to be landed. You should have write access and be able to land it.

**Pull request merged**